java/security/audit/weak-ssl-context.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
+- id: weak-ssl-context
+  metadata:
+    cwe: 'CWE-326: Inadequate Encryption Strength'
+    source_rule_url: https://find-sec-bugs.github.io/bugs.htm#SSL_CONTEXT
+  message: |
+    An insecure SSL context was detected. TLS versions 1.0, 1.1, and all SSL versions
+    are considered weak encryption and are deprecated.
+    Use SSLContext.getInstance("TLSv1.2") for the best security.
```

Comment:
Do you mean 1.3?

Reply:
From what I was reading, it isn't a guarantee that TLS 1.3 is available on every JDK implementation.

cf. https://stackoverflow.com/questions/29437596/tlsv1-3-is-it-available-now-in-java-8